packages/react-magma-docs/src/pages/api/combobox.mdx

```diff
@@ -913,6 +914,71 @@ interface CustomComboboxItem {
 }
 ```
 
+## Custom Item Render
+
```

Comment:
I feel like this could use a paragraph describing what this does (and how it's different from the other custom examples)

Reply:
Looks good!